hack/verify-goimports.sh

```diff
@@ -40,8 +40,8 @@ godirs=$(find . -not \( -path "./bin/*" -prune \) -name "*.go" | cut -d'/' -f2 |
 output=$($goimports $common_flags -l $godirs)
 
 if [ ! -z "${output}" ]; then
-    echo "${output}"
-    echo "+++ goimports failed; the following command may fix:" >&2
-    echo "+++ $goimports $common_flags -w $godirs" >&2
-    exit 1
+	echo "${output}" | sed "s/^/goimports: broken file: /"
```

Comment:
Praise: I ran `make verify-imports` with some broken goimports and the output is a lot clearer:

```
+++ running goimports
goimports: broken file: pkg/controller/acmechallenges/sync.go
+++ goimports failed; the following command may fix:
+++ /home/irbe/workspace/jetstack/cert-manager/bin/downloaded/tools/goimports@0.1.8  -w cmd devel hack internal make pkg test tools 
make: *** [make/ci.mk:12: verify-imports] Error 1
```